src/distributions/integer.rs

```diff
@@ -10,12 +10,10 @@
 
 use crate::distributions::{Distribution, Standard};
 use crate::Rng;
-#[cfg(all(target_arch = "x86", feature = "simd_support"))]
-use core::arch::x86::{__m128i, __m256i};
-#[cfg(all(target_arch = "x86_64", feature = "simd_support"))]
-use core::arch::x86_64::{__m128i, __m256i};
-use core::num::{NonZeroU16, NonZeroU32, NonZeroU64, NonZeroU8, NonZeroUsize,
-    NonZeroU128};
+#[cfg(target_arch = "x86")] use core::arch::x86::*;
+#[cfg(target_arch = "x86_64")] use core::arch::x86_64::*;
```

Comment:
We only want two items, right? I'm not so keen on using glob imports.

Reply:
4 items now. Added 2 `setzero` intrinsics

Reply:
True, though if you make the change below those will go away.